### PR TITLE
Prepare release for v0.32.0

### PR DESCRIPTION
**Pull request by romac**
Closes: #1313

### CODE REVIEW

File: .changelog/v0.32.0/summary.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+This release notably comes with a fully featured [light client attack detector][attack-detector],
```
Comment (thanethomson) [inline .changelog/v0.32.0/summary.md:1]:
```suggestion
*May 3, 2023*

This release notably comes with a fully featured [light client attack detector][attack-detector],
```

We actually also need to go back through previous releases' summaries and add release dates there too.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,54 @@
 # CHANGELOG
 
+## v0.32.0
+
+*May 2nd, 2023*
```
Comment (thanethomson) [inline CHANGELOG.md:5]: Was this line manually added? :slightly_smiling_face:

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,54 @@
...
+
+- [`tendermint-rpc`]: Export `Client` trait unconditionally, without
+  having to specify either the `http-client` or `websocket-client`
+  ([\#1235](https://github.com/informalsystems/tendermint-rs/issues/1235))
+- [`tendermint`]: Loosen bounds of merkle hashing functions to accept borrowed data.
+  ([\#1310](https://github.com/informalsystems/tendermint-rs/issues/1310))
+
 ## v0.31.1
 
+*April 17th, 2023*
```
Comment (thanethomson) [inline CHANGELOG.md:50]: If these were manually added, these changes would be clobbered upon regenerating the changelog. The better place to make the change is in each release's `summary.md` file.

File: README.md [hunk 4]
```diff
@@ -82,7 +84,7 @@ Papers
 
 ## License
 
-Copyright © 2020-2022 Informal Systems
+Copyright © 2020-2023 Informal Systems
```
Comment (thanethomson) [inline README.md:87]:
```suggestion
Copyright © 2020- Informal Systems and contributors
```

No need for an end date apparently. And, from what I understand, copyright belongs to whoever submitted the PR, so it's probably just worth mentioning contributors here.

**Review by thanethomson — COMMENTED** [review]
Nice! Just a few minor nits re: the changelog.

File: README.md (hunk 4 above)
Comment (tony-iqlusion) [reply README.md:87]: If you don't have a date range you might as well drop the hyphen too

```suggestion
Copyright © 2020 Informal Systems and contributors
```

File: .changelog/v0.32.0/summary.md (hunk 1 above)
Comment (romac) [reply .changelog/v0.32.0/summary.md:3]: https://github.com/informalsystems/tendermint-rs/pull/1314/commits/64a4904460872a3f05b7052dcef4d15d21bd4da6

File: CHANGELOG.md (hunk 2 above)
Comment (romac) [reply CHANGELOG.md]: Yes,

File: README.md (hunk 4 above)
Comment (romac) [reply README.md]: ddfc055

**Review by thanethomson — APPROVED** [review]

**Pull request merged**